### DISCUSSION TITLE
[gatsby-plugin-mdx] gives error after running query `html` in GraphQL

### DISCUSSION THREAD

**Discussion by Varniex** [gatsbyjs/gatsby#31484] [Help]
Whenever I start the development or build process then console shows the error
```
gatsby-plugin-mdx
[object Object]

Cannot find module .cache\caches\gatsby-plugin-mdx\webpack\output.js
```

My query is:

```
export const query = graphql`
    query {
        allMdx{
            nodes {
                html
                id
                frontmatter {
                    title
                }
            }
        }
    }
`;
```

Here is my package.json
```
"dependencies": {
    "@mdx-js/mdx": "^1.6.22",
    "@mdx-js/react": "^1.6.22",
    "gatsby": "^3.5.1",
    "gatsby-plugin-feed": "^3.5.0",
    "gatsby-plugin-image": "^1.5.0",
    "gatsby-plugin-mdx": "^2.5.1",
    "gatsby-plugin-react-helmet": "^4.5.0",
    "gatsby-plugin-sharp": "^3.5.0",
    "gatsby-plugin-theme-ui": "^0.8.4",
    "gatsby-remark-images": "^5.2.1",
    "gatsby-source-filesystem": "^3.5.0",
    "gatsby-transformer-sharp": "^3.5.0",
    "react": "^17.0.1",
    "react-dom": "^17.0.1",
    "react-helmet": "^6.1.0",
  }
```

Furthermore, if I run the same query in GraphQL environment, it doesn't display any output and my console shows this
```
Error: Conflict: Multiple chunks emit assets to the same filename output.js (chunks main and framework)
```

However, if I remove the `html` then it runs fine. 

I've been stuck in here for a couple of days. Any help would be appreciated. Thanks!!

Please let me know the further steps to resolve the issue.

**Comment by LekoArts** [comment]
The `html` property should only be used for RSS feeds really, in all other instances you want to query for `body` and pass that to the `MDXRenderer` as shown in our docs: https://www.npmjs.com/package/gatsby-plugin-mdx#mdxrenderer

**Reply by Varniex** [reply]
@LekoArts  Thanks for the answer!

I used `html` in `gatsby-plugin-feed` and it's showing similar error
```
gatsby-plugin-mdx
[object Object]

Error: The result of this StaticQuery could not be fetched.
```
However, if I am building the same config without `html` in query field in `gatsby-plugin-feed`, it is generating the feed without any error.

**Reply by talohana** [reply]
Facing the same issue. Did you use something else instead of `html` for `gatsby-plugin-feed`?

**Reply by Varniex** [reply]
Nothing extra. I did exactly given in the official documentation of `gatsby-plugin-feed`.
Although after removing `html` the plugin works fine but if I include `html` in query it's showing me error.

**Reply by hibouenverre** [reply]
Had exactly the same issue. Using the MDXRenderer fixed this issue for me, but also using the MarkdownRemark files instead of the MDX files in GraphQL worked.

**Reply by ghussain08** [reply]
Had the same issue for a couple of days, couldn't find any solution of 

```
Error: Conflict: Multiple chunks emit assets to the same filename output.js (chunks main and framework)
```

Removed `html` from  GraphQl query and it worked fine. Thank you very much @LekoArts .

**Reply by theowenyoung** [reply]
Same issue.

**Reply by vannyle** [reply]
I have the same issue. I use `html` only for `gatsby-plugin-feed`, and I get the warning.
```
warning gatsby-plugin-mdx
[object Object]
```
 Is there any solution for this case?

**Reply by Titou325** [reply]
Same issue here. No solution as far as I can tell, and using MDXRenderer for server-side rendering is very painful as the context is not loaded. Our use case here is that we generate the HTML from MDX to then store the data in Elasticsearch at build time. Must be processed server-side.

**Reply by ownchoice** [reply]
Having the same problem, the `html` property straight up doesn't work for MDX and I can't use it in any query, but works fine for markdown nodes (from another plugin).

Update: **Turns out, you can only query it on build, not on development mode o GraphiQL.** Source: https://github.com/gatsbyjs/gatsby/issues/17592#issuecomment-531173366

Issue tracking: https://github.com/gatsbyjs/gatsby/issues/17210

**Reply by geocine** [reply]
I understand that the `html` node was now removed from `gatsby-plugin-mdx`. I need the `html` for `gatsby-plugin-feed` , what are other options? Has anyone found a solution?
